sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -48,12 +48,12 @@ class ServiceBusAdministrationClient:  #pylint:disable=too-many-public-methods
 
     :param str fully_qualified_namespace: The fully qualified host name for the Service Bus namespace.
     :param credential: To authenticate to manage the entities of the ServiceBus namespace.
-    :type credential: Union[AsyncTokenCredential, ~azure.servicebus.aio.ServiceBusSharedKeyCredential]
+    :type credential: AsyncTokenCredential
     """
 
     def __init__(
             self, fully_qualified_namespace: str,
-            credential: Union["AsyncTokenCredential", ServiceBusSharedKeyCredential],
+            credential: Union["AsyncTokenCredential"],
```

Comment:
nit: we don't need "Union" here

Reply:
Fixed [here](https://github.com/Azure/azure-sdk-for-python/pull/13690)